src/context_store/dashboard/api_server.py

```
@@ -111,7 +120,7 @@ def main() -> None:
     app = create_app()
     uvicorn.run(
         app,
-        host="127.0.0.1",
+        host=settings.dashboard_host,  # noqa: B104
```

Ruff (0.15.9)

[warning] Invalid rule code in `# noqa`: B104

Add non-Ruff rule codes to the `lint.external` configuration option

(RUF102)